internal/extension/oop.go

```diff
@@ -214,31 +212,53 @@ func (p *OOPExtension) monitorStderr(stderr io.ReadCloser, stopChan <-chan struc
 	}
 }
 
-func (p *OOPExtension) logStderr(logLvl logLevel, logString string, err error) {
-	switch logLvl {
+func (p *OOPExtension) logStderr(logLine *oopLogEntry) {
+	keysAndValues := make([]any, 0, len(logLine.KeysAndValues)*2)
+	for key, value := range logLine.KeysAndValues {
+		keysAndValues = append(keysAndValues, key, value)
+	}
+	switch logLine.Level {
 	case LogLevelInfo:
-		p.logger.Info(logString)
+		p.logger.Info(logLine.Msg, keysAndValues...)
+	case LogLevelError:
+		p.logger.Error(fmt.Errorf("%v", logLine.Error), logLine.Msg, keysAndValues...)
 	default:
-		p.logger.Error(err, logString)
+		p.logger.Error(fmt.Errorf("unknown LogLevel %v", logLine.Level), logLine.Msg, keysAndValues...)
```

Comment:
Further log levels do exists, see [the slog](https://pkg.go.dev/github.com/go-logr/logr#FromContextAsSlogLogger) API